composer.json

```diff
@@ -32,7 +32,7 @@
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "gedmo/doctrine-extensions": "~2.4",
         "symfony-cmf/routing-bundle": "1.2.*",
-        "symfony/symfony": "~2.8.7, !=2.8.10",
+        "symfony/symfony": "^2.8.7, !=2.8.10 || ^3.0",
```

Comment:
Are you referring added there `"minimum-stability": "RC"` due symfony-cmf/routing-bundle 2.0 is not released yet? I thought as there's no minimum-stability set deps should resolve to stable versions, and in turn symfony should be resolved as ^2.8.7. Currently I'm working on project based on symfony 3.2 and integrating sulu for content management, but for now I'm testing it with symfony-cmf/routing-bundle 2.0.x-dev